components/tools/OmeroJava/test/integration/gateway/GatewayUsageTest.java

```diff
@@ -56,6 +60,24 @@ public void testLoginWithCredentials()
         gw.disconnect();
     }
 
+    /**
+     * Check logging in using a session ID.
+     * @throws DSOutOfServiceException unexpected
+     */
+    @Test
+    public void testLoginWithCredentialsSessionId()
+            throws DSOutOfServiceException {
+        final omero.client client =  new omero.client();
```

Comment:
I was just aiming for something straightforward and simple but maybe it's not sufficiently futureproof. Probably better then if I close this PR and create some larger scoping card and have @dominikl document the issue in the meantime?